docs/content/deployment/guides/kubernetes/deploying-with-helm-advanced.mdx

```diff
@@ -13,7 +13,7 @@ In this section, we demonstrate how to extend the [previous Helm deployment guid
 
 ## Prerequisites
 
-In addition to the [previous prerequisites](/deployment/guides/kubernetes/deploying-with-helm#prerequisites), we expect familiarity with \[Celery, a distributed task queue system]\(<https://docs.celeryq.dev/>).
+In addition to the [previous prerequisites](/deployment/guides/kubernetes/deploying-with-helm#prerequisites), we expect familiarity with [Celery, a distributed task queue system](<https://docs.celeryq.dev/>).
```

Comment:
Buildkite is throwing an error related to this line - could you address it? Run `make mdx-format` (replaces `make snapshot`)